manifests/c/Canonical/Ubuntu/2404/2404.0.5.0/Canonical.Ubuntu.2404.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Canonical.Ubuntu.2404
+PackageVersion: 2404.0.5.0
+PackageLocale: en-US
+Publisher: Canonical Group Limited
+PublisherUrl: https://canonical.com/
+PublisherSupportUrl: https://ubuntu.com/contact-us
+PrivacyUrl: https://ubuntu.com/wsl
+Author: Canonical Ltd. and the OpenSource community
+PackageName: Ubuntu 24.04
```

```suggestion
PackageName: Ubuntu 24.04 LTS
```

Should be the same as "Settings" panel.